packages/ecc-client-ga4gh-tes/src/components/tes-get-run/definition/tesGetRun.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  FASTElement,
+  attr,
+  customElement,
+  observable,
+} from '@microsoft/fast-element';
+import template from './tesGetRun.template.js';
+import styles from './tesGetRun.styles.js';
+import { fetchTask } from '../../../data/index.js';
+import { deleteTask } from '../../../data/Post/tesPost.js';
+import TaskData from './TaskData.js';
+
+@customElement({
+  name: 'ecc-tes-get-run',
+  template,
+  styles,
+  shadowOptions: { mode: 'open' },
+})
+export default class TESGetRun extends FASTElement {
+  @attr id = '';
+
+  @attr state = '';
+
+  @observable isLoading = true;
+
+  @observable data: TaskData = {
+    id: this.id,
+    state: this.state,
+    name: '',
+    description: '',
+    creation_time: '',
+    executors: [],
+    logs: [],
+  };
+
+  connectedCallback() {
+    super.connectedCallback();
+    // Add event listener to handle accordion open event
+    this.addEventListener('change', this.handleFetch);
+    const delButton = this.shadowRoot?.querySelector('fast-button');
+    if (!this.isLoading && delButton) {
+      // delButton.addEventListener("click", this.handleDelete);
+      delButton.addEventListener('click', this.handleDelete.bind(this));
+    }
+  }
+
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    this.removeEventListener('change', this.handleFetch);
+    const delButton = this.shadowRoot?.querySelector('fast-button');
+    if (!this.isLoading && delButton) {
+      delButton.addEventListener('click', this.handleDelete.bind(this));
```

Comment:
why are you adding an event listener on the disconnected callback?

Reply:
typo was supposed to be removeEventListener, vs code must have auto-corrected it, I'll fix it asap :+1:

Reply:
done